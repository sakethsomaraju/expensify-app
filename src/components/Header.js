import React from 'react' 
import {NavLink} from 'react-router-dom'

const Header = ()=>{
    return (
        <header>
            <h1>expensify</h1>
            <NavLink to="/" activeClassName="is-active" exact={true}>home </NavLink>
            <NavLink to="/create" activeClassName="is-active">create expense </NavLink>
            <NavLink to="/help" activeClassName="is-active">help </NavLink>

        </header>
    )
}

export default Header