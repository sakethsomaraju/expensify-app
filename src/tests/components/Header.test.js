import React from 'react'
import toJSON from 'enzyme-to-json'
import Header from '../../components/Header'
import {shallow} from 'enzyme'

test('Header should be rendered correctly',()=>{
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()

    const wrapper = shallow(<Header />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})