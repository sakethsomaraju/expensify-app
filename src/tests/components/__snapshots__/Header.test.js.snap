// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered correctly 1`] = `
<header>
  <h1>
    expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    home 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    create expense 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    help 
  </NavLink>
</header>
`;
